python/trifinger_simulation/collision_objects.py

```diff
@@ -57,43 +57,40 @@ def import_mesh(
     return obj
 
 
-class Block:
-    """
-    To interact with a block object
-    """
+class Cuboid:
+    """A cuboid which can be interacted with."""
 
     def __init__(
         self,
-        position=[0.15, 0.0, 0.0425],
-        orientation=[0, 0, 0, 1],
-        half_size=0.0325,
-        mass=0.08,
-        **kwargs,
+        position,
```

Comment:
I think it would be better to also have default options here, that way the instantiation would also be analogous to that of the Cube. Also, we could have the same name `half_width` in both classes.

Reply:
I intentionally did not set defaults as these arguments are all very application-dependent, so I don't think there are meaningful default values for them.  I only kept them in the Cube class to not break existing code.

Regarding `half_width`: I think "extents" is better for an arbitrary cuboid as to my understanding "width" only describes one of the three dimensions (the others being height and length).

Reply:
Yeah, that's true, I agree with that. It just gets a bit confusing while creating instances. About the former, yes, that's true too, default values just make debugging easier. Anyway, we could look into this later.